### ISSUE TITLE
Model IDLE Animation

### ISSUE THREAD

**Issue by sdilauro** [decentraland/godot-explorer#453]
There are different behaviors of the model animations

Foundation explorer:

https://github.com/user-attachments/assets/e9021b76-df49-4c48-999a-39f114019912

Mobile Godot Explorer:

https://github.com/user-attachments/assets/ffcb899a-361d-4495-9ead-6a38a861eb1c

**Comment by kuruk-mm** [comment]
I couldn't see the differences. Just the rock looks like different at the beginning. That what you are referring to? :thinking:

**Comment by sdilauro** [comment]
Yes, sorry. The rock is _broken_ (is the last frame of the 'mining' track) when should be idle
